woodwork/tests/accessor/test_accessor_utils.py

```diff
@@ -2,90 +2,19 @@
 import pytest
 
 from woodwork.accessor_utils import (
-    _get_valid_dtype,
     _is_dataframe,
     _is_series,
     get_invalid_schema_message,
     init_series,
     is_schema_valid
 )
 from woodwork.exceptions import TypeConversionError
-from woodwork.logical_types import (
-    Boolean,
-    Categorical,
-    Datetime,
-    NaturalLanguage
-)
 from woodwork.utils import import_or_none
 
 dd = import_or_none('dask.dataframe')
 ks = import_or_none('databricks.koalas')
 
 
-def test_init_series_valid_conversion_specified_ltype(sample_series):
```

Comment:
Even though this test and the three below it all use `_get_valid_dtype`, I think they are mainly testing `init_series`, so we probably want them to stay in `test_accessor_utils`

Reply:
Let's check with @thehomebrewnerd who suggested this change ([comment](https://github.com/alteryx/woodwork/pull/915#discussion_r632054205)) to get more details on which associated tests he's referring to.

Reply:
@jeff-hernandez Sorry, I wasn't very clear in my original comment. I'm thinking we should move `test_get_valid_dtype` since that is specifically testing we get the expected dtype value and that is no longer a utility function.  

We should leave the other `test_init_series*` tests where they were as they are really testing out that the `init_series` utility function behaves as expected in various circumstances.

Reply:
Okay, no problem. I'll update the tests.